sound/soc/sof/intel/hda-trace.c

```diff
@@ -19,16 +19,15 @@
 #include "../ops.h"
 #include "hda.h"
 
-static int hda_dsp_trace_prepare(struct snd_sof_dev *sdev)
+static int hda_dsp_trace_prepare(struct snd_sof_dev *sdev, struct snd_dma_buffer *dmab)
```

Comment:
why? because sdev is going to replaced? can you please add it to the commit message?

Reply:
I'm doing the conversion in two steps, so I need to support at this point the built in trace and the client version.